src/shared/utils/flattenChildren.js

```diff
@@ -22,22 +23,29 @@ var warning = require('warning');
  * @param {!string} name String name of key path to child.
  * @param {number=} selfDebugID Optional debugID of the current internal instance.
  */
-function flattenSingleChildIntoContext(traverseContext, child, name, selfDebugID) {
+function flattenSingleChildIntoContext(
+  traverseContext: mixed,
+  child: ReactElement<any>,
+  name: string,
+  selfDebugID: number
```

Comment:
Sorry, just saw this. `selfDebugID` is actually an optional parameter that will only be available in dev mode. You can also see from the `number=` JSDoc annotation. cc @vjeux